spec/db.test.js

```diff
@@ -0,0 +1,74 @@
+const {db, productQuery} = require('../database/index.js');
+const productList = require('../database/sample_products');
+const {profileImages, sampleImages, createImageUrl, createRandomQuantity} = require('../database/dbHelper.js');
+
+/*db helper function tests **************************************************************/
+
+test('the function createRandomQuantity takes in a maximum number and does not return the same number when called multiple times', () => {
+  const max = 10;
+  const randomNumber1 = createRandomQuantity(max);
+  const randomNumber2 = createRandomQuantity(max);
+  const randomNumber3 = createRandomQuantity(max);
+  const randomNumber4 = createRandomQuantity(max);
+  const randomNumbers = [randomNumber2, randomNumber2, randomNumber3, randomNumber4];
+
+  expect(randomNumbers.every((val) => val === randomNumber1)).toBe(false);
```

Comment:
If your numbers are random, wouldn't this have a chance to be true?

Reply:
Yeah, that's true.  I figured it was low enough probability.  Maybe there is a recursive solution that can do a better job.  I'll make a ticket for tech debt to work on that when I have a chance.